test/unit/controllers/footer.test.ts

```diff
@@ -30,9 +30,9 @@ describe('footer controller', () => {
     const routerGetStub: sinon.SinonStub = sandbox.stub(express.Router, 'get');
 
     setupFooterController();
-    expect(routerGetStub).to.have.been.calledWith(paths.footer.cookies);
-    expect(routerGetStub).to.have.been.calledWith(paths.footer.privacyPolicy);
-    expect(routerGetStub).to.have.been.calledWith(paths.footer.termsAndConditions);
+    expect(routerGetStub).to.have.been.calledWith(paths.common.cookies);
```

Comment:
maybe also add more structure here to `paths.common.footer.cookies`

Reply:
This is an example of what I am trying to say, paths.common.footer.cookies does not bring any info, cookies page is located inside the footer but it could not be the case...paths.common.cookies is IMO self-explanatory as well